app/controllers/user_controller.rb

```diff
@@ -0,0 +1,4 @@
+class UserController < ApplicationController
+  def new
+  end
+end
```

Comment:
Why is this controller created and why do you have a `new`method? Try running your tests without it.